gammapy/image/utils.py

```diff
@@ -137,11 +148,22 @@ def ring_correlate(image, r_in, r_out, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : array_like
+        Image to be correlated.
```

Comment:
Please no empty line between parameters here (you can fix this everywhere in this file while you're at it).